keras_cv/version_check_test.py

```diff
@@ -28,11 +39,11 @@ def test_check_tf_version_error():
 
 def test_check_tf_version_passes_rc2():
     # should pass
-    version_check.tf.__version__ = "2.9.1rc2"
+    tf.__version__ = "2.9.1rc2"
     version_check.check_tf_version()
 
 
 def test_check_tf_version_passes_nightly():
     # should pass
-    version_check.tf.__version__ = "2.10.0-dev20220419"
+    tf.__version__ = "2.10.0-dev20220419"
```

Comment:
Where are these and RC used?

Reply:
`tf.__version__` is checked in version_check.py, and this is just verifying that it works for nightly and rc builds